pkg/blocks/basicstep_test.go

```diff
@@ -61,18 +61,6 @@ steps:
   `,
 			wantError: false,
 		},
-		{
-			name: "Invalid basic",
```

Comment:
no longer relevant, as "does this have a name" is now validated in `step_test.go`